src/Redux/Books/Books.js

```diff
@@ -0,0 +1,18 @@
+export const ADD_BOOK = 'bookstore-cms/books/ADD_BOOK';
+export const REMOVE_BOOK = 'bookstore-cms/books/REMOVE_BOOK';
+export const initialState = [];
+
+/* eslint-disable default-param-last */
+function booksReducer(state = initialState, action) {
+  switch (action.type) {
+    case ADD_BOOK:
+      return [...state, action.book];
+    case REMOVE_BOOK:
+      return { ...state, items: state.filter((item) => item.id !== action.id) };
```

Comment:
- [ ] Your initial state is an array and therefore we need to remove the given book from the array and return the state as an array. hint;

```suggestion
    case REMOVE_BOOK:
      return state.filter(book => book.id !== id)};
```